docs/diagrams/resident-sequence-flow.iuml

```diff
@@ -0,0 +1,21 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/v2.2.0/C4_Deployment.puml
+!include https://raw.githubusercontent.com/LBHackney-IT/cv-19-res-support-v3/development/docs/diagrams/c4_shared.puml
+
+title Resident sequence flow diagram
+
+Resident -> Resident: Receive email
```

Comment:
Again this could be notification instead of email

Reply:
👍